distribution/zip/ballerina/src/assembly/bin.xml

```diff
@@ -209,6 +209,9 @@
                 <include>org.wso2.andes.wso2:andes-client:jar</include>
                 <include>org.wso2.securevault:org.wso2.securevault:jar</include>
 
+                <!-- Check-pointing dependencies -->
+                <include>com.google.code.gson:gson</include>
```

Comment:
We are working on developing our own serializer by extending the current json parser. This dependency will be removed after the new serializer. Please refer [1] .

[1] https://github.com/ballerina-platform/ballerina-lang/issues/9745